src/middlewared/middlewared/plugins/system.py

```diff
@@ -38,6 +38,8 @@
 FIRST_INSTALL_SENTINEL = '/data/first-boot'
 LICENSE_FILE = '/data/license'
 
+ECC_REGEX = re.compile(r'Total Width:\s*(\d*)')
```

Comment:
Not to be pedantic. but on every plugin we do this we start it with RE_  :P

Reply:
Updated :p